.github/workflows/test.yml

```diff
@@ -1,6 +1,9 @@
 name: gradido test CI
 
-on: push
+on:
+  push:
+    branches: 
+      - 2352-feat-user-story-user-authentication-reset-password
```

Comment:
remove

Reply:
Sure.
This will be removed, when everything in this PR is fine and mergeable.